upgrade-notes/backend_20250227_040635.md

```diff
@@ -0,0 +1,10 @@
+#### remove generated error pages ([#3827](https://github.com/shopsys/shopsys/pull/3827))
+
+- phing targets `error-pages-generate` and `test-error-pages-generate` were removed
+- `shopsys:error-page:generate-all` command was removed
+- Classes that handled errors were removed:
+    - `Shopsys\FrameworkBundle\Controller\Admin\ErrorController`
+    - `Shopsys\FrameworkBundle\Component\Error\` namespace
```

Comment:
That's not quite true as ErrorIdProvider was not removed. 

Let's just name those six classes

Reply:
Done